src/Caching/Hybrid/src/Runtime/HybridCache.cs

```diff
@@ -0,0 +1,119 @@
+using System;
```

Comment:
Missing license, build should complain?

Reply:
fixed; it did not complain - the collective "we" should check that the inspector is still working